nextflow_schema.json

```
@@ -394,7 +415,7 @@
                 },
                 "annotation_prokka_evalue": {
                     "type": "number",
-                    "default": 0.000001,
+                    "default": 1e-6,
```

```suggestion
                    "default": 0.000001,
```

Web-editor artificat, too...